### PR TITLE
Sloučená konfigurace linterů z toxu a pre-commitu na jedno místo

### LINKED ISSUE #131: tox -e lint a pre-commit
Aktuální stav je takový, že se nám lintery pouštějí v toxu a v pre-commit hooku. Nepraktické je, že máme i dvojí konfiguraci a bylo by fajn najít způsob, jak to sjednotit do jednoho.

Aktuálně je navíc pre-commit přísnější třeba na trailing whitespace než tox.

### PR DESCRIPTION
**Pull request by frenzymadness**
Fixes #131

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -16,14 +16,19 @@ repos:
     rev: 3.9.2
     hooks:
     -   id: flake8
+        additional_dependencies: [
+            "flake8-blind-except==0.2.0",
+            "flake8-debugger==4.0.0",
```
Comment (adelpopelkova) [inline .pre-commit-config.yaml]: I think, that the [debug-statements](https://github.com/pre-commit/pre-commit-hooks#debug-statements) hook from pre-commit does something similar. When you run the pre-commit check, that hook finds out, that there is a `breakpoint()` call used. I also tried using the `import pdb; pdb.set_trace()` and during this there are two checks that fail - one is the debug-statements one and another one is black which reformats it :smile:

Having said that, I am not against it, this one seems to have some added features.

File: requirements/lint.txt [hunk 2]
```diff
@@ -1,13 +1,4 @@
...
 
-# Security scanner
-bandit==1.7.0
-
-# Code checkers, linters and formatters
-black==21.6b0
-flake8==3.9.2
-flake8-blind-except==0.2.0
-flake8-debugger==4.0.0
-flake8-isort==4.0.0
```
Comment (adelpopelkova) [inline requirements/lint.txt:11]: This one isn't in the flake8 additional-dependencies, because `isort` is already a separate check?

**Review by adelpopelkova — COMMENTED** [review]
It looks good, it seems to be working, I had a few questions which are written lower.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (frenzymadness) [reply .pre-commit-config.yaml]: You are right. This one might be duplicate but, I'd say, without any evidence, that flake8 is Python-specific and therefore might be better. But I'm also okay with removing one of them.

File: requirements/lint.txt (hunk 2 above)
Comment (frenzymadness) [reply requirements/lint.txt:11]: Yes, exactly. `flake8-isort` depends on `isort` and just combine its capabilities with flake8 and that's something we do not need here.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (adelpopelkova) [reply .pre-commit-config.yaml]: The debug-statements one should be also Python specific, it just doesn't have _Python_ in the name.

Comment (frenzymadness) [reply .pre-commit-config.yaml]: You are right and from their code debug-statements seems to be more universal. Fixed.

**Review by adelpopelkova — APPROVED** [review]

**Pull request merged**